text/0000-fabric-private-chaincode-lite.md

```diff
@@ -247,27 +241,38 @@ This class includes smart contracts which operate on sensitive medical data, or
 
-***TODO: @Jeb can you please make pass on below and replace this section as appropriate with info from HBP, UMBC or alike***
-For example, Federated Learning on private sensitive information is a real-world use-case which FPC Lite enables securely on Hyperledger Fabric.
-To illustrate this, consider the case of training a model, e.g, Convolutional Neural Network (CNN), for detecting brain abnormalities such as precancerous lesions or aneurysms.
-To achieve high accuracy we need considerably more data than any single entity (e.g., a hospital) usually possess.
-Yet regulations like HIPAA make sharing brain CT scans or MRI studies, labeled by radiologists, hard if not impossible. Furthermore, to allow widest use of a derived model, it should be freely shareable without any privacy concerns. Lastly, to provide the necessary accountability and audit trail, such a federated application is ideally tied to a ledger. While there are cryptographic solutions to perform federated learning in a strongly private manner, e.g., based on differential privacy, for such a setting, they are very expensive in terms of computation and, in particular, communication complexity.
+Ideal use cases for FPC Lite involve operations on private data that do not involve the conditional revelation of those data. For example, Federated Learning on private sensitive information is a real-world use-case which FPC Lite enables securely on Hyperledger Fabric.
+In this use case, a decentralized group of organizations each possess private information – for example, hospitals possessing brain CT and MRI scans labeled by radiologists to indicate the presence of precancerous lesions.
+Regulations like HIPAA and GDPR make it difficult if not impossible to share these data; however, all can benefit from an AI model trained using all of their collective data.
+
+Federated Learning, as described in this original article: https://ai.googleblog.com/2017/04/federated-learning-collaborative.html, is a class of decentralized algorithms in which training is performed simultaneously across multiple nodes without the need to share training data.
+It provides some inherent privacy even without Trusted Execution, but with Trusted Execution is far more robust against various attacks in case one of the participants is compromised by an attacker or malicious insider.
+Specifically, FPC’s Trusted Execution Environment can assure that only vetted and unmodified training programs can participate in the algorithm, and that it includes mechanisms to foil possible attacks such as Adversarial Machine Learning.
+It can also keep a hardware-signed record of model and training data provenance that can be used to audit the system in case bad data is introduced.
+Through each training job, a model is produced which does not incorporate any of the training data from which it was created; the model is published to the group without any revelation of the private training data.
```

Comment:
Hmm, i don't like the replacement of PATE with the google approach you mention. it has ill (non) defined privacy and just slapping TEEs around doesn't make it well defined. The PATE had a very well-defined security, but required a trusted party which perfectly fits FPC. And the resulting privacy is the best you can get ...